src/client/src/orangehrmAdminPlugin/pages/languagePackage/languagePackage.vue

```diff
@@ -0,0 +1,188 @@
+            icon-name="plus"
+            display-type="secondary"
+            @click="onClickAddLanguage"
+          />
+        </div>
+      </div>
+      <table-header :total="total"></table-header>
+      <div class="orangehrm-container">
+        <oxd-card-table
+          v-model:selected="checkedItems"
+          v-model:order="sortDefinition"
+          :headers="headers"
+          :items="items?.data"
+          :selectable="false"
+          :clickable="false"
+          :loading="isLoading"
+          row-decorator="oxd-table-decorator-card"
+        />
+      </div>
+      <div class="orangehrm-bottom-container">
+        <oxd-pagination
+          v-if="showPaginator"
+          v-model:current="currentPage"
+          :length="pages"
+        />
+      </div>
+    </div>
+    <br />
+    <add-language-modal
+      v-if="showAddLanguageModal"
+      @close="onAddLanguageModalClose"
+    >
+    </add-language-modal>
+  </div>
+</template>
+<script>
+import {computed, ref} from 'vue';
+import usePaginate from '@ohrm/core/util/composable/usePaginate';
+import {navigate} from '@ohrm/core/util/helper/navigation';
+import {APIService} from '@/core/util/services/api.service';
+import useSort from '@ohrm/core/util/composable/useSort';
+import AddLanguageModal from '@/orangehrmAdminPlugin/components/AddLanguageModal.vue';
+
+const defaultFilters = {
+  languageName: '',
+};
+
+const defaultSortOrder = {
+  languageName: 'ASC',
+};
+
+export default {
+  name: 'LanguagePackageList',
+
+  components: {
+    'add-language-modal': AddLanguageModal,
+  },
+
+  setup() {
+    const filters = ref({...defaultFilters});
+
+    const {sortDefinition, sortOrder, onSort} = useSort({
+      sortDefinition: defaultSortOrder,
+    });
+
+    const serializedFilters = computed(() => {
+      return {
+        sortOrder: sortOrder.value,
+      };
+    });
+
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      '/api/v2/admin/i18n/languages',
+    );
+
+    const {
+      showPaginator,
+      currentPage,
+      total,
+      pages,
+      pageSize,
+      response,
+      isLoading,
+      execQuery,
+    } = usePaginate(http, {
+      query: serializedFilters,
+    });
+
+    onSort(execQuery);
+
+    return {
+      http,
+      showPaginator,
+      currentPage,
+      isLoading,
+      total,
+      pages,
+      pageSize,
+      execQuery,
+      items: response,
+      filters,
+      sortDefinition,
+    };
+  },
+  data() {
+    return {
+      showAddLanguageModal: false,
+      headers: [
+        {
+          name: 'name',
+          slot: 'title',
+          title: 'Language Packages',
+          sortField: 'languageName',
+          style: {flex: 2},
+        },
+        {
+          name: 'actions',
+          slot: 'footer',
+          title: this.$t('general.actions'),
+          cellType: 'oxd-table-cell-actions',
+          cellConfig: {
+            translate: {
+              onClick: this.onClickDelete,
```

Comment:
should be `onClickTranslate` and since there is no method for this. let's remove for now

Reply:
Done